resources/views/external-link-confirm.blade.php

```diff
@@ -100,14 +100,24 @@ function initExternalLinkConfirm() {
         links.forEach(function (link) {
             link.setAttribute('data-external-link-confirm', 'true');
 
-            link.addEventListener('click', function(e) {
+            const clickHandler = (e) => {
                 if (localStorage.getItem('has_disabled_link_warning') === 'true') {
                     return;
                 }
 
                 e.preventDefault();
 
                 Livewire.emit('openModal', 'external-link-confirm', link.getAttribute('href'));
+            }
+
+            link.addEventListener('click', clickHandler);
+
+            link.addEventListener('mousedown', (e) => {
```

Comment:
issue with `mousedown` is that it triggers immediately, and not after "clicking". Wouldn't something like https://developer.mozilla.org/en-US/docs/Web/API/Element/auxclick_event be more suitable to keep it in line with the regular click handling? 🤔

Reply:
nvm no full support for that event in all browsers :sadup:

still think it feels a bit odd having the middle click open it immediately as opposed to the regular click, will check for alternatives